janitor/functions.py

```diff
@@ -55,6 +57,78 @@
 )
 
 
+def get_symbol(symbol: str) -> str:
+    """
+    This is a helper function to get a companies full
+    name based on the stock symbol.
+
+    Example:
+        print(get_symbol("aapl"))
+        console >> Apple Inc.
+
+    :param symbol: This is our stock symbol that we use
+        to query te api for the companies full name.
+    :return: This is the company name
+    """
+    result = requests.get(
+        "http://d.yimg.com/autoc."
+        + "finance.yahoo.com/autoc?query={}&region=1&lang=en".format(symbol)
+    ).json()
+
+    for x in result["ResultSet"]["Result"]:
+        if x["symbol"] == symbol:
+            return x["name"]
+        else:
+            return "Not found."
+
+
+def is_connected(url: str) -> bool:
+    """
+    This is a helper function to check if the client
+    is connected to the internet.
+
+    Example:
+        print(is_connected("www.google.com"))
+        console >> True
+
+    :param url: We take a test url to check if we are
+        able to create a valid connection.
+    :return: We return a boolean that signifies our
+        connection to the internet
+    """
+    try:
+        sock = socket.create_connection((url, 80))
+        if sock is not None:
+            sock.close()
+            return True
+    except OSError:
+        pass
```

Comment:
I think I'm lacking some context. Under what circumstances will an OSError show up?

Following Pythonic practice, I might be better to raise the error explicitly and loudly rather than let it silently pass. What do you think of the following pattern instead?

```suggestion
    except OSError as e:
        import warnings
        warnings.warn("There was an issue connecting to the internet. Please see original error below.")
        raise e
```

Reply:
**Action item:** Please address the questions above. We will discuss further on what the most optimal pattern here ought to look like.